### PR TITLE
fixed images test from not working. GetAll test will be implemented l…

### PR DESCRIPTION
**Pull request by Jaylapuck**
…later down the line since I am currently puzzled as to how to do it.

### CODE REVIEW

**Review by JordanAlbayrak — APPROVED** [review]
LG2M

File: Calligraphy.Test/Form/FormControllerTests.cs [hunk 1]
```diff
@@ -179,40 +220,75 @@ public async void PostBadRequestTestNoEmail()
...
+            var pagedResponse = new PagedResponse<IEnumerable<FormEntity>>(dummyForms, dummyFilter.PageNumber,
+                dummyFilter.PageSize);
+            
+            _mockFormService.Setup(x => x.GetAll(dummyFilter, dummyRoute )).Returns(new OkObjectResult(pagedResponse));
+            
+            // Act
+            var result = _formController.GetAllPages(dummyFilter);
+            
+            Assert.IsType<OkObjectResult>(result);
+            */
```
[the remark this answers is not in the document]
Comment (Jaylapuck) [reply Calligraphy.Test/Form/FormControllerTests.cs:291]: yes, later on I will fix this piece of code and wanted to keep it in order to not  have to write it again

[the remark this answers is not in the document]
Comment (Trim0500) [reply Calligraphy.Test/Form/FormControllerTests.cs:291]: Fair enough

**Review by Trim0500 — APPROVED** [review]
I'll approve this!

**Pull request merged**